paymentsheet/src/main/java/com/stripe/android/paymentsheet/paymentdatacollection/ach/USBankAccountFormViewModel.kt

```diff
@@ -342,7 +342,8 @@ internal class USBankAccountFormViewModel @Inject internal constructor(
                                 ApiRequest.Options(
                                     apiKey = lazyPaymentConfig.get().publishableKey,
                                     stripeAccount = lazyPaymentConfig.get().stripeAccountId
-                                )
+                                ),
+                                expandFields = emptyList()
```

Comment:
@jameswoo-stripe we don't need the payment method to be expanded on payment sheet right?

Reply:
No, we don't need to expand the payment method 👍